docs/doc_start/15_basics.rst

```diff
@@ -138,7 +138,7 @@ The skeleton shown in the graphic above is defined in a file ``person.py`` which
 
 That's it. When this Skeleton is connected to a module later on, ViUR's admin tools like the Vi automatically provide an auto-generated input mask on it.
 
-A Skeleton does automatically provide the bone ``key`` also, which is an instance of the class :class:`baseBone<core.bones.bone.baseBone>`. This bone holds the value of the unique entity key, that is required to uniquely identify an entity within the database. The pre-defined bones ``creationdate`` and ``changedate`` of each skeleton store the date and time when the entity was created or changed. In terms of ViUR, an entity is a document or dataset in the datastore, that stores information.
+A Skeleton does automatically provide the bone ``key`` also, which is an instance of the class :class:`keyBone<core.bones.bone.keyBone>`. This bone holds the value of the unique entity key, that is required to uniquely identify an entity within the database. The pre-defined bones ``creationdate`` and ``changedate`` of each skeleton store the date and time when the entity was created or changed. In terms of ViUR, an entity is a document or dataset in the datastore, that stores information.
```

Comment:
Use upper-case class names only. It's now `KeyBone`, `BaseBone`, `StringBone`, etc.